### PR TITLE
[CBRD-24204] applyinfo usage is modified

### PR DESCRIPTION
**Pull request by eido5**
[http://jira.cubrid.org/browse/CBRD-24204](http://jira.cubrid.org/browse/CBRD-24204)

the utility "applyinfo" emits usage. 
-L option is required to execute the -a and -p options.
So, `the -L option is required` is added into the usage.

As mentioned closed PR, previous PR is invalid and then it is closed.
And I applied usage as like comment.
If we accept this modification, I will apply to utils.msg in the `en_US.utf8`, `km_KH.utf8` and `vi_VN.utf8`

AS-IS
```
applyinfo: display CUBRID HA Apply information.
usage: cubrid applyinfo [OPTION] database-name
valid options:
  -r, --remote-host-name      remote host name; display remote node's active log information
  -a, --applied-info          display applied information
  -L, --copied-log-path=PATH  path of copied log volumes; display copied log information
  -p, --page=ID               page id; default : 0
  -v, --verbose               enable verbose status messages; default : disable
  -i, --interval=S            print information every S secs

```

TO-BE
```
applyinfo: display CUBRID HA Apply information.
usage: cubrid applyinfo [OPTION] database-name
valid options:
  -r, --remote-host-name      remote host name; display remote node's active log information
  -a, --applied-info          display applied information; the -L option is required
  -L, --copied-log-path=PATH  path of copied log volumes; display copied log information
  -p, --page=ID               page id; the -L option is required; default : 0
  -v, --verbose               enable verbose status messages; default : disable
  -i, --interval=S            print information every S secs

```

**Implementation**

N/A
**Remarks**

N/A

### CODE REVIEW

**Review by hornetmj — APPROVED** [review]

**Comment by hornetmj** [comment]
@eido5 How about changing the target branch from CUBRID:CBRD-24204_msg to develop?

File: msg/en_US/utils.msg [hunk 1]
```diff
@@ -978,9 +978,9 @@ usage: %1$s applyinfo [OPTION] database-name\n\
 \n\
 valid options:\n\
   -r, --remote-host-name      remote host name; display remote node's active log information \n\
-  -a, --applied-info 	      display applied information \n\
+  -a, --applied-info 	      display applied information; the -L option is required \n\
   -L, --copied-log-path=PATH  path of copied log volumes; display copied log information \n\
```
Comment (joohok) [inline msg/en_US/utils.msg]:
```suggestion
  -L, --copied-log-path=PATH  path of copied log volumes; display copied log information \n\
  -a, --applied-info 	      display applied information; the -L option is required \n\
```

how about change the sequence ? 
It seems good to first describe the L option, and then describe the a and p options that depend on the L option.

**Comment by eido5** [comment]
> @eido5 How about changing the target branch from CUBRID:CBRD-24204_msg to develop?

I agree. I will try it. Thank you

File: msg/en_US/utils.msg (hunk 1 above)
Comment (eido5) [reply msg/en_US/utils.msg]: I think so. It will be more readable. Thank you

**Review by joohok — APPROVED** [review]

**Review by wrlawodms — COMMENTED** [review]
Looks good. Please re-request review after having other languages supported.

**Comment by eido5** [comment]
> Looks good. Please re-request review after having other languages supported.

I will re-request, when msg files are modified in the other languages and those files are committed. Thank you.

**Comment by eido5** [comment]
@hornetmj @wrlawodms @joohok 
14 language msg files are changed.
Review please.

File: msg/ko_KR.euckr/utils.msg [hunk 2]
```diff
@@ -291,7 +291,7 @@ backupdb: 
   -t, --thread-count=COUNT     ���� �����ϴ� ������ ����; �⺻��: �ڵ�\n\
       --no-compress            ��� ����� �������� �ʽ�ϴ�; �⺻��: ����\n\
       --sleep-msecs=N          �� 1M �б� ���� N �и��ʸ� ���ϴ�; �⺻��: 0\n\
-  -k, --separate-keys          TDE�� ���Ǵ� Key File (_keys)� ���� �������� �ʰ� ������ ���Ϸ� �и��մϴ�.\n
+  -k, --separate-keys          TDE�� ���Ǵ� Key File (_keys)� ���� �������� �ʰ� ������ ���Ϸ� �и��մϴ�. \n
```
Comment (wrlawodms) [inline msg/ko_KR.euckr/utils.msg]: unrelated

**Review by wrlawodms — APPROVED** [review]

File: msg/ko_KR.euckr/utils.msg (hunk 2 above)
Comment (eido5) [reply msg/ko_KR.euckr/utils.msg]: @wrlawodms 
Right. Actually , I transformed from msg/**ko_KR.utf8**/utils.msg to msg/**ko_KR.euckr**/utils.msg. 

Because I cannot write down broken characters like "����" in the msg/ko_KR.euckr/utils.msg. 

So, I think this code should be **not changed** and applyinfo usage should be **changed** in this case. Right?

Comment (eido5) [reply msg/ko_KR.euckr/utils.msg]: @wrlawodms 
Unrelated code is rolled back. I will register about previous code as a issue.

**Review by hornetmj — APPROVED** [review]

**Review by joohok — APPROVED** [review]

**Comment by eido5** [comment]
@ssihil The coverage is failed. 
Can I request that identifying the failure? (which occurs because the "applyinfo" is fixed.)

+)In this PR, the sentence `the -L option is required` is added in the usage.

**Comment by hornetmj** [comment]
@mhoh3963 The coverage test complains about this PR. But, it is not related to it when I check. So, please I ask you to merge this PR after confirming.
